app/logic/userManagement.py

```diff
@@ -33,3 +33,27 @@ def changeCurrentTerm(term):
     newCurrentTerm.save()
 
     session["current_term"] = model_to_dict(newCurrentTerm)
+
+def addTerm():
+    terms_table = {"Spring":"Summer",
+                    "Summer":"Fall",
+                    "Fall":"Spring"}
+    terms = list(Term.select().order_by(Term.id))
+    lastCreatedTerm = terms[-1]
+    term = lastCreatedTerm.description
```

Comment:
maybe rename this variable to termDescription

Reply:
Fixed